src/index.js

```diff
@@ -37,14 +38,15 @@ class FileIncludeWebpackPlugin {
 
   process(compilation, callback) {
     const { context, output } = this.compiler.options
+    const outputPath = this.outputPath !== undefined ? this.outputPath : output.path;
     this.context = path.join(context, this.source)
     const files = utils.getRequiredFiles(this.context, '')
 
     utils.logger.info(`Working on ${files.length} .html files`)
 
     files.forEach(file => {
       const sourcePath = path.join(this.context, file)
-      const destinationPath = path.join(output.path, file)
+      const destinationPath = path.join(outputPath, file)
```

Comment:
change this to 
`const destinationPath = this.destination ? path.join(this.destination, file) : file`